common/build/eslint-config-fluid/minimal.js

```diff
@@ -264,6 +264,9 @@ module.exports = {
         "@typescript-eslint/prefer-includes": "error",
         "@typescript-eslint/prefer-nullish-coalescing": "error",
         "@typescript-eslint/prefer-optional-chain": "error",
+
+        // By default libraries should not take dependencies on node libraries
+        "import/no-nodejs-modules": "warn",
```

Comment:
Might be worth expanding the comment to note that packages intended to run in a node environment are welcome to turn this off.